javascript/javascript1/week2/script.js

```diff
@@ -0,0 +1,167 @@
+// Flight booking full name function =============================================================================
+
+const getFullName = (firstName, lastName) => `${firstName} ${lastName}`;
+
+const fullName1 = getFullName('Rostyslav', 'Zhyvitskyi');
+const fullName2 = getFullName('Benjamin', 'Hughes');
+
+console.log(fullName1, fullName2);
+
+// Formal full name
+
+const getFullNameFormal = (firstName, lastName, sex, useFormalName) => {
+  if (firstName === '' || lastName === '') {
+    throw Error('Please make sure you have entered your first and last name');
+  }
+
+  if (useFormalName) {
+    return sex === 'male'
+      ? `Mr. ${firstName} ${lastName}`
+      : `Mrs. ${firstName} ${lastName}`;
+  } else {
+    return `${firstName} ${lastName}`;
+  }
+};
+
+// Event application ===============================================================================================
+
+const weekDays = [
```

Comment:
Should it be global or local?

Reply:
Weekdays I think can be global, because we can use them in many places (not in this project, but in general) and they are the same for everybody.

Reply:
True, but in this particular file/project, should it be global?